packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -63,8 +54,6 @@ const MainTitleBar = () => {
                     <div className='main-title-bar-mobile--account-type-dropdown'>
                         <AccountTypeDropdown />
                     </div>
-                    {/* TODO: This is for testing purposes only */}
```

Comment:
@sergei-deriv same question here

Reply:
@mahdiyeh-deriv Same answer, but just one case for `Desktop` and 2nd is for `Mobile`

Reply:
@sergei-deriv we need to update the logic and connect it to mock server and then remove this You can do it here but it's out of the scope of the card.

Reply:
@mahdiyeh-deriv It's just a button which switches flag to show/hide popup, it's not related to conditions to show/hide this popup, this button was added just for QA team when they tested this popup UI

Reply:
@sergei-deriv We are calling the `setWalletsMigrationFailedPopup` action when clicking on that  button, need to put the action call somewhere else to make sure the logic will not be missed later on